ports/psoc6/boards/manifest.py

```diff
@@ -0,0 +1,6 @@
+freeze("$(PORT_DIR)/freeze")
+# include("$(MPY_DIR)/extmod/uasyncio")
```

Comment:
Remove all these comments? nothing of this is needed or?

Reply:
ok